concepts/0003-protocols/roles-participants-etc.md

```diff
@@ -1,83 +1,41 @@
-# Roles, Participants, Parties, and Controllers
+### Roles, Participants, Parties, and Controllers
 
-## Roles
+#### Roles
 
-The __roles__ in a protocol are the perspectives (responsibilities, privileges) that parties take on an
-interaction. 
+The __roles__ in a protocol are the perspectives (responsibilities, privileges) that parties take i an
+interaction.
 
 This perspective is manifested in three general ways:
 
  * by the expectations that a party takes on in a protocol (ex. a role may be expected to do something to start a protocol).
  * by the messages that a party can and does use in the course of the protocol (some messages may be reserved for a single role, while other may used by some if not all roles).
  * by the state and the transition rules
- 
-Like parties, roles are normally known at the start of the protocol but this is not a requirement.
 
-In an auction protocol, there are only two roles--*auctioneer*
-and *bidder*--even though there may be many parties involved.
+Like parties, roles are normally known at the start of the protocol but this is not a requirement.
 
-## Participants
+#### Participants
 
-The __participants__ in a protocol are the agents that consume and/or emit
-[plaintext application-level messages](
-https://github.com/hyperledger/indy-hipe/tree/master/text/0026-agent-file-format#agent-plaintext-messages-ap)
-that embody the protocol's interaction. Alice, Bob, and
-Carol may each have a cloud agent, a laptop, and a phone; if they engage in an
-introduction protocol using phones, then the agents on their phones are the participants.
-If the phones talk directly over Bluetooth, this is particularly clear--but even if the
-phones leverage push notifications and HTTP such that cloud agents help with routing,
-only the phone agents are participants, because only they maintain state for the
-interaction underway. (The cloud agents would be __facilitators__, and the laptops would
-be __bystanders__). When a protocol is complete, the participant agents know about the
-outcome; they may need to synchronize or replicate their state before other agents of the
-parties are aware.
+The __participants__ in a protocol are the agents that send and/or receive
+[plaintext application-level messages](../../concepts/../features/0044-didcomm-file-and-mime-types/README.md)
+that embody the protocol's interaction.
 
-## Parties
+#### Parties
 
 The __parties__ to a protocol are the entities directly responsible for achieving the protocol's goals.
 When a protocol is high-level, parties are typically people or organizations; as protocols become lower-level,
-parties may be specific agents tasked with detail work through delegation.
```

Comment:
I don't like losing this example. This is a concept RFC, not a impl spec; we need to explain the concepts well. The definition that remains is accurate, but too terse.

Reply:
I find this kind of example very confusing in trying to understand the core concepts and so detract from the overall understanding.

Reply:
Disagree.

Reply:
You disagree that I find it confusing?  :-)

Reply:
Reverted

Reply:
I'm not disagreeing that you find it confusing; obviously you're the world authority on that assertion. :-) I'm disagreeing with the judgment call that says, "take it out, because the confusion is getting in the way of the goal." I'm claiming that if the example doesn't make sense, it's because of a deep problem in understanding. The example must be grappled with. If it can't be processed comfortably, then subtle misunderstandings remain. Therefore, a better change would be to rewrite the section such that it makes perfect sense to you -- not to delete it.